CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+## Types of Contributions
+
+### Report Bugs
+
+Report bugs at https://github.com/hackebrot/pytest-cookies/issues
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+### Fix Bugs
+
+Look through the GitHub issues for bugs. Anything tagged with "bug"
+and "help wanted" is open to whoever wants to implement a fix for it.
+
+### Implement Features
+
+Look through the GitHub issues for features. Anything tagged with "enhancement"
+and "help wanted" is open to whoever wants to implement it.
+
+### Write Documentation
+
+pytest-cookies could always use more documentation, whether as part of the
+official docs, in docstrings, or even on the web in blog posts, articles, and such.
+
+### Submit Feedback
+
+The best way to send feedback is to file an issue at https://github.com/hackebrot/pytest-cookies/issues.
+
+If you are proposing a new feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+## Get Started!
+
+Ready to contribute? Here's how to set up `pytest-cookies` for local development. Please note this documentation assumes
+you already have (virtualenv)https://virtualenv.pypa.io/en/stable/installation and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Gi) installed and ready to go.
+
+1\. Fork the `pytest-cookies` repo on GitHub.
+
+2\. Clone your fork locally:
+
+``` bash
+$ cd path_for_the_repo
+$ git clone git@github.com:YOUR_NAME/pytest-cookies.git
+```
+
+3\. Assuming you have virtualenv installed (If you have Python3.5 this should already be there), you can create a new environment for your local development by typing:
+
+``` bash
+$ virtualenv pytest-cookies-env
+$ source pytest-cookies-env/bin/activate
+```
+This should change the shell to look something like `(pytest-cookies-env) $``
+
+4\. Create a branch for local development:
+
+``` bash
+$ git checkout -b name-of-your-bugfix-or-feature
+```
+
+Now you can make your changes locally.
+
+5\. When you're done making changes, check that your changes pass flake8. Since, this package contains mostly templates the flake should
+   be run for tests directory:
+
+``` bash
+$ flake8 ./tests
```

Comment:
Please change this to ``$ tox -e flake8``.

This will make sure that everyone runs checks against the same files. (Tox also runs flake8 against ``setup.py`` and ``pytest_cookies.py``)